tickets/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls    import path 
+
+from tickets.views import FlightLocationView
+
+urlpatterns = [
+    path('', FlightLocationView.as_view()),
```

Comment:
그냥 ""이렇게 하게 되면 url이 /flights가 되는데 이는 비행 정보를 달라는 의미에 더 가깝습니다.
```suggestion
    path('/locations', FlightLocationView.as_view()),
```